frontend/src/v5/ui/controls/dueDate/dueDateInput.component.tsx

```diff
@@ -15,10 +15,10 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
This name should be <code>dueDate<del>Input</del>.component.tsx</code>
Sorry, I only just saw this.